### DISCUSSION TITLE
微信配置之后，使用echo,通过http调用测试没问题，但是会话无返回

### DISCUSSION THREAD

**Discussion by wzj1223** [pwh-pwh/aiwechat-vercel#29] [Q&A]
公众号聊天会话无返回，测试chat接口是没问题的

**Comment by pwh-pwh** [comment]
可能是WX_TOKEN配置错了，要看下vercel的日志信息

**Reply by wzj1223** [reply]
<img width="1334" alt="image" src="https://github.com/pwh-pwh/aiwechat-vercel/assets/23468503/9672dc13-1cbe-40b2-81e2-37479123dc09">
我看调用chat可以的，但是会话不行，我看配置没错呀

**Reply by wzj1223** [reply]
![image](https://github.com/pwh-pwh/aiwechat-vercel/assets/23468503/b8fe7b16-20d1-43be-a045-2fdda54b7cc2)
<img width="680" alt="image" src="https://github.com/pwh-pwh/aiwechat-vercel/assets/23468503/bd2c24df-4f98-4ac3-874e-031cc9d68244">

**Reply by pwh-pwh** [reply]
问题复杂超时了
